src/kernel_2.jl

```diff
@@ -1,27 +1,40 @@
 function key_sampling(
-  g::SimpleWeightedGraph{T,R}
-  ) where T<:Integer where R<:Real
+  g::Graph{T}
+  ) where T<:Integer
 
   n_v = nv(g)
-  rand_keys = sample(1:n_v, 64, replace = false)
+  rand_keys = randperm(n_v)
```

Comment:
I have to make sure that I don't include a key twice for bfs. So if I don't get a 64 vertices in the first run of sample I might end up with same vertex in second run of sample. Then I will also have to make sure that in subsequent runs of sample I don't end up including the same vertices again. Can you please suggest a workaround for this ?